webserver/Models/BZEAccount.cs

```diff
@@ -1,7 +1,12 @@
 using Microsoft.AspNetCore.Identity;
+using NuGet.Common;
 
 namespace webserver.Models;
 
 public class BZEAccount : IdentityUser {
     public DateTime LastLogin { get; set; }
+
+    public BZEAccount(){
+        LastLogin = DateTime.UtcNow;
```

Comment:
Every time you instantiate this class, the LastLogin will be updated.
I don't think you want that.

If you are searching for a user, this value will be updated.

Reply:
The idea is that this class is only instantiated when we are creating a user
Perhaps it's wrong to assume a created user has ever logged in, though

This value is updated when we log in, though